go.mod

```diff
@@ -152,5 +154,6 @@ require (
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/tharsis/ethermint => /Users/danielburckhardt/code/tharsis/ethermint
```

Comment:
why replace?

Reply:
just noticed this too. Was still left over from debugging. I just pushed the correction